src/labels/CustomLabels.labels

```diff
@@ -6057,6 +6057,30 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
         <shortDescription>Failure message if an invalid base64 url was provided</shortDescription>
         <value>The base64url format is invalid. It can only contain these characters: [a-z A-Z 0-9 _ -]</value>
     </labels>
+    <labels>
+        <fullName>gePaymentProcessErrorGeneric</fullName>
+        <categories>Gift Entry</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Generic failure message description for tokenization or purchase errors</shortDescription>
+        <value>There was an error processing your payment. Please review your payment information and try again. Alternatively, you can select Do Not Charge Card to enter the pledged donation without payment processing.</value>
+    </labels>
+    <labels>
+        <fullName>gePaymentProcessingErrorBanner</fullName>
+        <categories>Gift Entry</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Banner error message for payment processing error.</shortDescription>
+        <value>There was an error processing the payment.</value>
+    </labels>
+    <labels>
+        <fullName>gePaymentServicesErrorResponse</fullName>
+        <categories>Gift Entry</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Error message from payment services.</shortDescription>
```

Comment:
```suggestion
        <shortDescription>Error message from {0}.</shortDescription>
```

Reply:
@dfuller114 I don't suppose it's possible to use a variable in a shortDescription, is it? If so, it might be safer to substitute "Elevate payment processing" with a variable pointing to the commonPaymentServices custom label.

Reply:
fixed